apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/image-block-keywords/index.js

```diff
@@ -0,0 +1,24 @@
+/* global wp */
+
+/**
+ * External dependencies
+ */
+import { assign } from 'lodash';
+
+const additionalKeywords = [ 'logo', 'brand', 'emblem', 'hallmark' ];
+
+wp.hooks.addFilter(
+	'blocks.registerBlockType',
+	'full-site-editing/editor/image-block-keywords',
+	( settings, name ) => {
+		if ( name !== 'core/image' ) {
+			return settings;
+		}
+
+		settings = assign( {}, settings, {
+			keywords: settings.keywords.concat( additionalKeywords ),
```

Comment:
Not a blocker, but is it better to spread the values into a new array rather than mutating the original just in case object reference changes are needed for change detection at any point in the process?
```suggestion
			keywords: [ ...settings.keywords, ...additionalKeywords ],
```